draft-ietf-privacypass-auth-scheme.md

```diff
@@ -140,23 +140,31 @@ in more detail.
 ## Token Challenge {#challenge}
 
 Origins send a token challenge to clients in an "WWW-Authenticate" header field
-with the "PrivateToken" scheme. This challenge includes a TokenChallenge
-message, along with information about what keys to use when requesting a token
-from the issuer.
+with the "PrivateToken" scheme. This field is a series of key-value pairs, where
+values are base64url-encoded binary data. Each field has two mandatory parameters,
```

Comment:
Not all parameters to the auth scheme will necessarily be base64... I wonder if we shouldn't go into the key-value bit, since that's normal to an auth scheme.